sdk/communication/communication-network-traversal/src/models.ts

```diff
@@ -12,4 +13,19 @@ export interface CommunicationRelayClientOptions extends PipelineOptions {}
 /**
  * GetRelayConfiguration operation options for the request
  */
-export interface GetRelayConfigurationOptions extends OperationOptions {}
+export interface GetRelayConfigurationOptions extends OperationOptions {
+  /**
+   * The user for whom to issue a token
+   */
+  id?: string;
+
+  /*
+   * The specified routeType for the relay request
+   */
+  routeType?: RouteType;
+
+  /*
+   * The specified Time to live for the relay credential
```

Comment:
```suggestion
   * The specified time to live for the relay credential in seconds
```
It's important that we mention the unit at least in the comment.